src/components/infoTooltips/FrozenTooltip.tsx

```diff
@@ -0,0 +1,51 @@
+import { ExclamationIcon } from '@heroicons/react/outline';
+import { Trans } from '@lingui/macro';
+import { Box, SvgIcon } from '@mui/material';
+
+import { ContentWithTooltip } from '../ContentWithTooltip';
+import { Link } from '../primitives/Link';
+import { frozenProposalMap } from '../../utils/marketsAndNetworksConfig';
+
+interface FrozenTooltipProps {
+  symbol?: string;
+  currentMarket?: string;
+}
+
+export const getFrozenProposalLink = (
+  symbol: string | undefined,
+  currentMarket: string | undefined
+): string => {
+  if (currentMarket && currentMarket === 'proto_harmony_v3') {
+    return 'https://snapshot.org/#/aave.eth/proposal/0x81a78109941e5e0ac6cb5ebf82597c839c20ad6821a8c3ff063dba39032533d4';
+  } else if (currentMarket && currentMarket === 'proto_fantom_v3') {
+    return 'https://snapshot.org/#/aave.eth/proposal/0xeefcd76e523391a14cfd0a79b531ea0a3faf0eb4a058e255fac13a2d224cc647';
```

Comment:
Small optimization if you wanted to, you could assign the baseUrl to a var for DRY